providers/squadcast/escalation_policy.go

```diff
@@ -0,0 +1,80 @@
+package squadcast
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+)
+
+type EscalationPolicyGenerator struct {
+	SquadcastService
+	teamID string
```

Comment:
Only one field is exported. let's either export both or none at all.

Reply:
fixed after making use of `Request` function in `squadcast_service.go` file